.github/workflows/lint.yaml

```diff
@@ -54,6 +54,7 @@ jobs:
               uses: shivammathur/setup-php@v2
               with:
                   php-version: '8.2'
+                  extensions: ':psr'
```

Comment:
If we need to do this, this require a dev-kit PR

Reply:
See shivammathur/setup-php#799.
I guess we should wait to confirm if the extension is disabled in that project or implement the solution on our side.